src/helpers/generateFile.ts

```diff
@@ -25,7 +25,9 @@ export const generateFile = (
 export type Generated<T> = T extends ColumnType<infer S, infer I, infer U>
   ? ColumnType<S, I | undefined, U>
   : ColumnType<T, T | undefined, T>;
-export type Timestamp = ColumnType<Date, Date | string, Date | string>;`;
+export type Timestamp = ColumnType<Date, Date | string, Date | string>;
+export type EnumArrayInner<T extends string, All extends string> = T extends infer Single extends string ? T | \`\${Single},\${Exclude<All, Single>}\` : never
+export type EnumArray<T extends string> = '{}' | \`{\${EnumArrayInner<T, T>}}\`;`;
```

Comment:
having this always would throw ts error for unused variable, maybe would be good to add this conditionaly.